loadbalancer/fadein_test.go

```diff
@@ -76,15 +76,15 @@ func testFadeIn(
 			})
 		}
 
-		hashKeys := findHashKeys(a, ctx)
+		rnd := rand.New(rand.NewSource(time.Now().UnixNano()))
 		t.Log("test start", time.Now())
 		var stats []string
 		stop := time.After(fadeInDuration)
-		// Emulate the load balancer loop, sending requests to it with iterative hash keys
-		// of every endpoint over and over again till fadeIn period is over.
+		// Emulate the load balancer loop, sending requests to it with random hash keys
+		// over and over again till fadeIn period is over.
 		func() {
 			for {
-				ctx.Params[ConsistentHashKey] = hashKeys[len(stats)%len(hashKeys)]
+				ctx.Params[ConsistentHashKey] = strconv.Itoa(rnd.Intn(100000))
```

Comment:
The old version of test code creates the limited amount of keys: one key for one host, so basically in case of 6 hosts (3 "old" ones: `a, b, c` and 3 "fading-in": `d, e, f`) this code will send request with the following key values in the loop:
```
h_a
h_b
h_c
h_d
h_e
h_f
h_a
h_b
... (~100k reqs for single test)
```

As in the beginning of this test, nodes `d, e, f` process very few amount of requests (like here https://github.com/zalando/skipper/pull/2363#discussion_r1223171116), instead such requests are redirected to the corresponding host in the consistent hash circle (https://www.toptal.com/big-data/consistent-hashing). 
For example, requests to **d** may be redirected to **e**, **e** - to **f**, **f** - to **a**.
In the old version this corresponding is constant, because the same six keys are used, so there is high chance that host **d** will receive not so many requests.

So, the solution I propose here is to use different keys for the forementioned redirection to be not constant, but random instead.